capstone/target/portfolio-1/script.js

```diff
@@ -159,3 +159,13 @@ function displayPanel(name) {
   document.getElementById("panel").style.display = "block";
   document.getElementById("restaurant-info").innerHTML = name;
 }
+
+function getSmallBizNames() {
+  fetch('/gimmeSmallBizNames').then(response => response.json()).then((restaurantNames) => {
+    for(let name of restaurantNames) {
+      console.log(name);
```

Comment:
Why are we committing the console log? I'm okay with it as long as there's a reason it's there to help further development. That said, there should be an issue attached for the plan to remove it.

Reply:
just wanted to make sure i was getting good outputs. yeah gonna put an issue to remove it and replace with insert names into hashset.